src/app/hooks.ts

```diff
@@ -0,0 +1,5 @@
+import { AppDispatch, RootState } from './store';
+import { useDispatch, useSelector } from 'react-redux';
+
+export const useAppDispatch = useDispatch.withTypes<AppDispatch>();
+export const useAppSelector = useSelector.withTypes<RootState>();
```

Comment:
The `useDispatch` and `useSelector` hooks from `react-redux` do not have a `withTypes` method. Instead, you should create typed versions of these hooks by using TypeScript generics. For example, `export const useAppDispatch = () => useDispatch<AppDispatch>();` and `export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;`.